apps/prairielearn/src/pages/instructorIssues/instructorIssues.html.ts

```diff
@@ -100,7 +100,7 @@ export function InstructorIssues({
             <div class="card-header bg-primary text-white">
               <div class="d-flex flex-row align-items-center mb-2">
                 <div class="d-flex flex-column">
-                  <h1 class="h6 font-weight-normal mb-0">Issues</h1>
+                  <h1>Issues</h1>
```

Comment:
Gonna guess you didn't test this:

<img width="452" alt="Screenshot 2024-08-16 at 10 43 41" src="https://github.com/user-attachments/assets/b1997d8a-49bf-44f6-b069-a57c30e9b2ab">

See my note here: https://github.com/PrairieLearn/PrairieLearn/pull/10485#discussion_r1719057190

Reply:
You would be correct. Added the classes back (and tested this time) in [c5187bb](https://github.com/PrairieLearn/PrairieLearn/pull/10483/commits/c5187bb79e07a5a99f75ef5cfa5a9961d6781e2d).